tests/Unit/CssInlinerTest.php

```diff
@@ -2002,7 +2000,7 @@ public function inlineCssWithValidMinifiedMediaQueryContainsInnerCss(string $css
 
         $subject->inlineCss($css);
 
-        self::assertStringContainsString('<style type="text/css">' . $css . '</style>', $subject->render());
+        self::assertContainsCss('<style type="text/css">' . $css . '</style>', $subject->render());
```

Comment:
`assertContainsCss` expects only the CSS declaration, but not the `style` node, if I understand this correctly. Or am I mistaken?

Reply:
It can cope with the `<style>` element as well - e.g. https://github.com/MyIntervals/emogrifier/blob/main/tests/Support/Constraint/CssConstraint.php#L32-L33.

Mostly it would just allow some whitespace differences within it, unless any parts of it (such as attribute values) looked very much like parts of CSS where variation is catered for.

Reply:
PS. There's a data provider [`provideEquivalentCssInStyleTags`](https://github.com/MyIntervals/emogrifier/blob/main/tests/Support/Traits/CssDataProviders.php#L142) used in tests such as [`StringContainsCssTest::matchesHaystackContainingNeedle`](https://github.com/MyIntervals/emogrifier/blob/main/tests/Unit/Support/Constraint/StringContainsCssTest.php#L21-L31).

Reply:
Okay, thanks! It looks like we should improve the documentation in that regard. I've created #1075 to track this.